cumulusci/cumulusci.yml

```diff
@@ -863,7 +863,7 @@ flows:
                 flow: dependencies
                 options:
                     update_dependencies:
-                        include_beta: True
+                        resolution_strategy: preproduction
```

Comment:
Are you sure we want to use the preproduction resolution strategy by default, even for flows that are oriented toward development? It means that if I'm a partner developing an extension to NPSP, I'll get the beta versions of NPSP and its underlying packages, which I probably don't want.

Or is this just a temporary situation maintaining the status quo until you've had a chance to get the PRs in to change the default resolution strategy for our internal repositories?

Reply:
Never mind, I continued reading and realized preproduction is an alias for latest_release by default.